libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -896,7 +896,7 @@ bool RC_Channel::run_aux_function(aux_func_t ch_option, AuxSwitchPos pos, AuxFun
     const bool ret = do_aux_function(ch_option, pos);
 
     // @LoggerMessage: AUXF
-    // @Description: Auixillary function invocation information
+    // @Description: Auxillary function invocation information
```

Comment:
should this be "Auxiliary"? Looks like you got rid of the wrong letter
```suggestion
    // @Description: Auxiliary function invocation information
```

Reply:
yup, it's a single L